crates/aptos/src/move_tool/mod.rs

```diff
@@ -50,10 +51,14 @@ pub enum MoveTool {
 impl MoveTool {
     pub async fn execute(self) -> CliResult {
         match self {
-            MoveTool::Compile(tool) => to_common_result(tool.execute().await),
-            MoveTool::Publish(tool) => to_common_result(tool.execute().await),
-            MoveTool::Run(tool) => to_common_result(tool.execute().await),
-            MoveTool::Test(tool) => to_common_result(tool.execute().await),
+            MoveTool::Compile(tool) => {
```

Comment:
nit: refactor to the suggestion above? e.g.,

```
let result = match self {...};
to_common_result(result).await
```

Reply:
Same, types!